apps/examples/heap_stat_realtime/Kconfig_ENTRY

```diff
@@ -0,0 +1,3 @@
+config ENTRY_HEAP_STAT_REALTIME
```

Comment:
Hi,

Please add below Kconfig header as we follow in TizenRT.

```
#
# For a description of the syntax of this configuration file,
# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
#
```

Reply:
I didn't see the same header in other Kconfig_ENTRY files.
I will add this header in all Kconfig_ENTRY files if necessary.